eras/conway/impl/CHANGELOG.md

```diff
@@ -13,6 +13,12 @@
   * `reCurrentEpochL`
   * `reCommitteeStateL`
 
+### `testlib`
+
+  * Removed `redelegateDRep` from `ImpTest`
+  * Changed signature of `delegateToDRep` to take a `Credential` parameter
```

Comment:
Technically those are breaking changes, but I highly doubt that anyone uses these functions, so we can ignore that fact :smile:

Reply:
I was thinking the same, but In the meantime I introduced another breaking change by changing Set to Nonempty in the predicate failure, so I'll change this now to 1.17.0.0  to reflect the breaking change.